README.md

```diff
@@ -15,7 +15,7 @@ The container that has the code that configures the packet forwarder's region an
 
 https://github.com/NebraLtd/hm-pktfwd
 
-## Gateway-config
+## Gateway Config
```

Comment:
This was deliberately written like this as the idea is that each matches the name of the actual docker container.